### PR TITLE
Add API examples and descriptions for Traceability DTOs

### PR DESCRIPTION
**Pull request by wjpurdum**
Add examples and descriptions for Issuer API DTOs.

### CODE REVIEW

File: packages/traceability/issuer-api/src/pods/certificate/dto/certificate.dto.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class CertificateDTO {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     blockchain?: any;
 
-    @ApiProperty({ type: String })
+    @ApiProperty({ type: String, example: "x8888"})
```
Comment (josipbagaric) [inline packages/traceability/issuer-api/src/pods/certificate/dto/certificate.dto.ts]: Transaction hashes are usually in a different format:
[0x2b8da531e46cff1e217abc113495befac9384339feb10816b0f7f2ffa02fadd4](https://explorer.energyweb.org/tx/0x2b8da531e46cff1e217abc113495befac9384339feb10816b0f7f2ffa02fadd4)

File: packages/traceability/issuer-api/src/pods/certificate/dto/tx-hash.dto.ts [hunk 2]
```diff
@@ -2,6 +2,6 @@ import { ApiProperty } from '@nestjs/swagger';
 import { ContractTransaction } from 'ethers';
 
 export class TxHashDTO {
-    @ApiProperty({ type: String })
+    @ApiProperty({ type: String, example: "x99999999"})
```
Comment (josipbagaric) [inline packages/traceability/issuer-api/src/pods/certificate/dto/tx-hash.dto.ts]: Same here as above.

File: packages/traceability/issuer-api/src/pods/certificate/dto/claim-data.dto.ts [hunk 3]
```diff
@@ -1,29 +1,29 @@
 import { ApiProperty } from '@nestjs/swagger';
 import { IClaimData } from '@energyweb/issuer';
 import { IsDateString, IsString } from 'class-validator';
-
+//TO DO
```
Comment (ioncreature) [inline packages/traceability/issuer-api/src/pods/certificate/dto/claim-data.dto.ts]: todo :)

**Review by ioncreature — APPROVED** [review]

File: packages/traceability/issuer-api/src/pods/certificate/dto/certificate.dto.ts [hunk 4]
```diff
@@ -11,33 +11,33 @@ import {
 } from 'class-validator';
 import { ClaimDTO } from './claim.dto';
 import { EnergyDTO } from './energy.dto';
-
+//LOOK FOR EXAMPLE
```
Comment (josipbagaric) [inline packages/traceability/issuer-api/src/pods/certificate/dto/certificate.dto.ts]: :)

**Comment by josipbagaric** [comment]
@ioncreature the build also fails on DeepScan for something unrelated. Could you check what's going on?

**Comment by ioncreature** [comment]
@JosephBagaric yep, I'll fix it

**Review by ioncreature — APPROVED** [review]

**Pull request merged**